src/app/(full-layout)/transit/fetchSubway.ts

```
@@ -0,0 +1,103 @@
+
+  for (let i = 0; i < MAX_KEY_INDEX; i++) {
+    const apiKey = API_KEYS[i]
+    try {
+      const response = await fetch(
+        `http://swopenapi.seoul.go.kr/api/subway/${apiKey}/json/realtimeStationArrival/1/20/%EB%8B%B9%EC%82%B0`,
+      )
+      const res = await response.json()
+
+      // 호출 횟수 초과 에러 발생 시
+      if (res.message && res.message.includes(ERROR_MESSAGE)) {
+        console.log(`API_KEY_${i}: 호출 횟수 초과`)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)